loops-and-arrays.js

```diff
@@ -11,6 +11,12 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
```

Comment:
De aanpak om "je" aan elke naam toe te voegen is eenvoudig en effectief. De lus is duidelijk en goed leesbaa